### ISSUE TITLE
Testset does not contain every target label

### ISSUE THREAD

**Issue by MRsAIbot** [TobiasMR/emotion-recognition-from-posture#1]
The problem is that the subsampled testset does not contain every label. I subsample the dataset into a test and training set in the following way

``` python
from sklearn import cross_validation

X_train, X_test, Y_train, Y_test = cross_validation.train_test_split(
        X, Y, test_size=0.25, random_state=0)
```

However, there are few occurrences of the label F in the original dataset. We find that after subsampling

``` python
print set(Y_test)
print set(Y_train)
```

results in

``` python
set(['C', 'D', 'T'])
set(['C', 'F', 'D', 'T'])
```
